### ISSUE TITLE
Enforce english as governing language for github project

### ISSUE THREAD

**Issue by TwizzyDizzy** [Sonerezh/sonerezh#96]
Hi folks,

this is not meant as an insult or something, but I think it would really contribute to the spreading and the interest of developers in sonerezh, if the governing language here at github would be english.

This would make it easier for all people to check, whether their bug is already reported (because the don't get it, if they wouldn't copy the issue text over to translate.google.com) or help others with their issues.

This issue in itself can be closed as soon as reviewed, maybe future issues that are not in english could be cross-referenced to this issue and reporters could be asked to translate their issue to english.

Cheers
Thomas

**Comment by peterbrinck** [comment]
I would really love that too!!
+1 :+1:

**Comment by nodiscc** [comment]
In my experience (maintaining a project with a - relatively - large french-speaking user base), it is best if maintainers can reply to questions in foreign languages in english. Properly explaining bugs/feature requests in english is not easy for everyone. Maintainers can translate the report to english after it's posted. getting clear feedback in french is better than no feedback at all.

**Comment by TwizzyDizzy** [comment]
Yeah, I've thought about that, too. It's a trade-off, for sure. I think at least the headline could be translated by github maintainers so that when scrolling throught the issues one could guess whether the headline goes roughly in the direction of ones own issue.

Cheers
Thomas

**Comment by lGuillaume124** [comment]
Hi,
In fact we did not expect that Sonerezh will be downloaded all over the world, it's a very small project ^^
Your suggestions are always interesting and I agree with @nodiscc, it's always easier to describe a bug using your native language **but** English is essential if we want to work with everyone.

So from now on I will try to translate all requests from French to English :)
